tests/unit-tests/test-class-lesson.php

```diff
@@ -101,13 +100,110 @@ public function testIsPreRequisiteComplete() {
 
 	}
 
+	public function testGetCourseId() {
```

Comment:
I think it would be nice to add some comments to every test. I think it's a practice we're using in Sensei codebase for new PHP tests.

Reply:
Nice! Added some very basic documentation in [2014fc4](https://github.com/Automattic/sensei/pull/5294/commits/2014fc48a697c22bbfbec4019e8f2c88c87087ee). Hope it's...ok? :)

Reply:
That's perfect! I think we could just add the period after the descriptions. 😉

Reply:
Done in [1f4fc6a](https://github.com/Automattic/sensei/pull/5294/commits/1f4fc6afcce4e8bce49166a39d35409433868ff7). Thanks!